.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
Add this to gitignore.

Reply:
@WaleedAshraf actually, I committed it on purpose, IMO should be part of project with multiple participants: https://editorconfig.org/

Please confirm, if you want to remove it :)

Reply:
@WaleedAshraf removed, it could be create anytime, if needed 👍